app.json

```diff
@@ -7,7 +7,7 @@
     "icon": "./assets/icon.png",
     "splash": {
       "image": "./assets/splash.png",
-      "resizeMode": "contain",
+      "resizeMode": "cover",
```

Comment:
what does this do? is this related to iPad sizing?

Reply:
this is for the splash screen. it makes sure that there's no ugly white background when it appears @sukritsangvong